spec/factories/item.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :item do
+  end
```

Comment:
Question, No edit needed - If you specify a factory with no parameters, does that mean you are creating an item with all of its fields set to nil? How does this work?

Reply:
It is essentially an empty object that you can specify parameters when you create the new instance. You can create a new object without specifying any parameters. 

Update: I made the change to have default parameters that can be overwritten when you create a new item. I also added validations and validation tests for the model.